main.cc

```diff
@@ -1440,7 +1440,17 @@ struct add_wiring_tool : tool
     }
 
     void get_description(char *str) override {
-        sprintf(str, "Place wiring type %u", type);
+        switch (type) {
+        case wire_type_power:
+            sprintf(str, "Place wiring type %s", "power");
```

Comment:
They don't need to be sprintf, but since this is intended to be a temporary fix until it is replaced by a lookup table, I felt consistency between the different cases was worth the minor efficiency hit, as the eventual solution will need to use sprintf.